specification/communication/data-plane/CallAutomation/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the communicationservices.
 
 ```yaml
 openapi-type: data-plane
-tag: package-2025-03-30-preview
+tag: package-2025-06-15
```

Comment:
It looks like this was based on 2025-03-30-preview, but shouldn't it be based on the more recent GA, 2025-05-15?

Reply:
Hi @mikekistler , thanks for your question. Just to clarify: the diff shows an update from 2025-03-30-preview, but the actual base is GA 2025-05-15. 2025-03-30-preview was listed previously because it was released after the earlier GA, but we’re incorporating its changes on top of 2025-05-15. The versions were just released out of sequence.

Reply:
Sorry we missed this when we reviewed 2025-03-30-preview. We (all of us) need to avoid this kind of thing going forward.